### DISCUSSION TITLE
Double / Tripple Announcement v2.2.1 and v2.2.2

### DISCUSSION THREAD

**Discussion by JensenNick** [JensenNick/huawei_solar_pees#82] [Announcements]
First of all with the release v2.2.1, the *”Huawei Solar PEES package”* now automatically fetches information about your inverter model and sets the rated power accordingly.

Secondly (and thirdly) with  release v2.2.2, the *”Huawei Solar PEES package”* now partially support the new inverter models LC0, MAP0 and MB0 and on top of this the package also support the corresponding iStore products for the Australian users.

The input file has been split up into one related to PEES and one related to STAT.

A huge thanks to @Roving-Ronin for his comprehensive input to these updates.

**Comment by heinoskov** [comment]
Nice work @JensenNick
